brianlong/www.validators.app
# 180228159_make_website_url_a_link

#### What's this PR do?
- Make validator url a link

#### How should this be manually tested?
- Go to validator show page, check details sections with Website row - if url is present then you should see a link that should open new tab

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/180228159

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
